pixi_browse/tui/version_loader.py

```
@@ -143,6 +170,52 @@ async def load_version_details(
 
+        artifact_data = await self.load_version_artifact_data(
+            package_name,
+            record,
+            preview_key=preview_key,
+        )
+        about_urls = self.about_urls_cache.get(preview_key, AboutUrls())
+        run_exports: RunExportsJson | None = None
+        try:
+            run_exports = await self.get_run_exports(str(record.url))
+        except Exception:
+            pass
```

Ruff (0.15.6)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)